litestar/contrib/jwt/jwt_token.py

```diff
@@ -42,6 +43,8 @@ class Token:
     """Audience - intended audience."""
     jti: str | None = field(default=None)
     """JWT ID - a unique identifier of the JWT between different issuers."""
+    _extras: dict[str, Any] | None = field(default=None)
```

Comment:
Sorry if a dumb question - looking at this without any context. Are the extra fields intended to be accessed by users in any way?

Also, can you use `default_factory=dict` and not require the optional union for the type?

Reply:
Yep, I can do the `default_factory` for sure.  

Re: the first question.  The answer is maybe. The JWT may contain any arbitrary claims, but they may only be used in certain contexts or may not be relevant for the application.  

As an example, [here](https://developers.google.com/identity/sign-in/web/backend-auth) is what the JWT token for an IAP authenticated request from Google.  These can change over time and could be different based on the defined oauth scopes.  

In my case, I only need 1 of the extra fields, but the bigger issue is that an exception is raised when I try to create a `Token` class when it has these extra fields.

Reply:
> but the bigger issue is that an exception is raised when I try to create a Token class when it has these extra fields

Understood. Basically I'm querying whether the `_extra` attribute should have the leading underscore or not, as that implies to users that they shouldn't be accessing the collection.

Reply:
It probably should not have the leading underscore since the user may need to get to these attributes.